src/Raven.Server/ServerWide/SecretProtection.cs

```diff
@@ -669,6 +677,9 @@ public RavenServer.CertificateHolder LoadCertificateFromPath(string path, string
                 // we need to load it as exportable because we might need to send it over the cluster
                 var loadedCertificate = new X509Certificate2(rawData, password, X509KeyStorageFlags.Exportable | X509KeyStorageFlags.MachineKeySet);
 
+                // make sure that we are using supported encryption format by bouncy castle
+                rawData = loadedCertificate.Export(X509ContentType.Pfx, password);
```

Comment:
Shouldn't this be:

```suggestion
                rawData = loadedCertificate.Export(X509ContentType.Pfx);
```

Reply:
What's the difference?

Reply:
You are exporting encrypted with password or not. In memory, we just want to use the unencrypted one. Simplify things